log/logtest/recorder.go

```diff
@@ -86,6 +90,10 @@ func (r *Recorder) addChildLogger(nr *Recorder) {
 
 // Enabled indicates whether a specific record should be stored
 func (r *Recorder) Enabled(ctx context.Context, record log.Record) bool {
+	if r.enabledFn == nil {
+		return defaultEnabledFn(ctx, record)
+	}
```

Comment:
This leads to a race condition e.g. when `Enabled` is concurrently called for zero value of `Recorder`.

Reply:
It shouldn't, as we only read the data from the struct, we don't write to it.
I've added a call to `Enabled()` in the concurrent safe test.

Reply:
You are right.